config/locales/states/en.yml

```diff
@@ -2417,16 +2401,16 @@ en:
           has_assistance: I hereby state that I am unable to sign my application for a mail-in ballot without assistance because I am unable to write by reason of my illness or physical disability. I have made or have received assistance in making my mark in lieu of my signature.
           Address of witness: Address of witness or assistant
         abr:
-          request_deadline: "Received by 5pm on October 26, 2021."
+          request_deadline: "received by 5:00PM on the Tuesday before the election"
           abr_status_check_intro: "You can look up your Mail in ballot status by using your state's online tool:"
           state_reminder_1: "If you received help with this form, make sure that the witness has also signed the form and included their address."
           state_reminder_2: ""
-          finalpage_instructions: "You can look up your mail-in ballot status by using Pennsylvania's <a href=\"https://www.pavoterservices.pa.gov/pages/ballottracking.aspx\">Election Ballot Status</a> page"
+          finalpage_instructions: "You can look up your mail-in ballot status by using Pennsylvania's <a href="https://www.pavoterservices.pa.gov/pages/ballottracking.aspx">Election Ballot Status</a> page"

```

Comment:
@laurenrtv please don't make these types of changes (removing a \ before a ") unless you see it there in the live page. These are a necessary part of the formatting.